target/linux/realtek/dts-5.15/rtl8393_hpe_1920.dtsi

```diff
@@ -0,0 +1,246 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "rtl839x.dtsi"
+#include "rtl83xx_hpe_1920.dtsi"
+
+#include <dt-bindings/leds/common.h>
+
+/ {
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: led-0 {
+			label = "green:power";
+			color = <LED_COLOR_ID_GREEN>;
+			function = LED_FUNCTION_POWER;
+			gpios = <&gpio0 9 GPIO_ACTIVE_HIGH>;
+		};
```

Comment:
What about the `Mode` and `RPS` leds?

Reply:
> What about the `Mode` and `RPS` leds?

Do you know which gpio those are connected to? Most other realtek switches seem to have a single status LED.

Reply:
No. But in the GPL source dump there are some references: https://svanheule.net/switches/gpl/hpe-1910-uboot.tar.gz
Inside of `v2r1_copyright/board/smb/hp1910_48/smbhp1910_48.c` there is a reference to `GPIO 15` as alarm led.
I suspect this could be a red led. References to the correct GPIOs for `Mode` and `RPS` will most likely only be in the official firmware and not in uboot.

Reply:
Hmm. Without a clear fix would rather get this merged and work on adding extra LEDs later.

Reply:
@Hurricos any luck with the LEDs on the other GPIOs?